src/Skills.jsx

```diff
@@ -193,19 +233,14 @@ class Skills extends Component {
               onClick={this.openSkillModal} >
               Add Skill
             </Button>
-
             <Modal
               isOpen={this.state.skillModalIsOpen}
               onRequestClose={this.closeSkillModal}
               contentLabel="SkillModal"
-              style={ModalStyle}
             >
               <SkillForm api={api}
-                         closeModal={this.closeSkillModal}
-                         reloadSkills={this.reloadSkills}
```

Comment:
CloseSkillModal is still passed to `SkillForm` but `reloadSkills` will not be, as it isn't called anywhere in `SkillForm` anyways.